File: net/src/Sdl.Web.GraphQL/Response/IGraphQLResponse.cs
using System.Collections.Generic;
using Sdl.Web.HttpClient;

namespace Sdl.Web.GraphQLClient.Response
{
    public interface IGraphQLResponse
    {
        dynamic Data { get; set; }
        List<GraphQLError> Errors { get; set; }
        HttpHeaders Headers { get; set; }
    }
}

File: java/src/main/java/com/sdl/web/pca/client/response1/IGraphQLResponse.java
package com.sdl.web.pca.client.response;

import java.util.List;

public interface IGraphQLResponse {
    Object Data = null;
    List<GraphQLError> Errors = null;

    void setData(Object data);
    Object getData();

    void setErrors(List<GraphQLError> errors);
    List<GraphQLError> getErrors();
}
